Cargo.toml

```diff
@@ -26,15 +26,22 @@ wasmtime-wasi = { git = "https://github.com/bytecodealliance/wasmtime.git", rev
 wasi-common = { git = "https://github.com/bytecodealliance/wasmtime.git", rev = "8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306", default-features = false }
 env_logger = "0.7"
 log = "0.4"
+tokio = { version = "0.2", features = ["macros"] }
+warp = { version = "0.2", features = ["tls"] }
+
+serde = { version = "1.0", features = ["derive"] }
+serde_yaml = "0.8"
+serde_json = "1.0"
+serde_derive = "1.0"
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
This appears to only be used for certificate generation. However, `warp` (above) pulls in the `rustls` crypto chain. I'd prefer not to have two dependencies that do the same thing. So let's avoid using `openssl` for now.

Reply:
I thought we wanted to be using openssl for FIPS futures?  The main reason, however, is that I struggled to find instructions for how to generate certificates - the only way I managed to make it work was with openssl.  Happy to be given pointers...

Reply:
@MikeCamel `warp` only currently supports the `rustls` stack. So while FIPS is definitely a future concern, my more immediate concern is the size bloat of having two separate crypto stacks.

As far as certificate generation... there doesn't seem to be a lot of options. The `rustls` stack uses `webpki` which only provides validation. The `x509-parser` project only appears to deserialize the structures.

I see a couple options:

1. Accept multiple crypto stacks. I don't love this option.

2. Use an http framework that supports openssl. There don't seem to be any. One potential route would be to use `hyper` directly via `hyper-tls`. However, `hyper-tls` currently only supports the client side. So we'd have to patch it to support the server side.

Thoughts? @ueno Your input is valued here.

Reply:
3. We could create an X.509 template as bytes that we serialize just key data into. This is lightweight and avoids another crypto library.